master/internal/api_workspace.go

```diff
@@ -85,8 +85,6 @@ func (a *apiServer) GetWorkspaceProjects(ctx context.Context,
 		return nil, err
 	}
 
-	a.sort(resp.Projects, req.OrderBy, req.SortBy,
```

Comment:
question: Does this still work even with this `a.sort` removed?

Reply:
My new understanding is that `a.sort` is for objects which aren't sorted by the database for some reason (for example, it's used on checkpoints and in notebooks, but never on GetExperiments).
e2e_tests include sorting by id and name to verify orders that workspaces and projects are returned